### DISCUSSION TITLE
"quire new" error on npm temp directories

### DISCUSSION THREAD

**Discussion by mpopke** [thegetty/quire#747] [Get Help] [answered]
Hi all.

I have a fresh install of Quire (v1), and I can't seem to get the quire new command to work. Command and output below:

```
% quire new russell
[CLI:quire] init-starter 
  project root: "russell" 
  starter: "https://github.com/thegetty/quire-starter-default"
russell set to use quire-11ty@latest
[CLI:quire] installing @thegetty/quire-11ty@latest into russell
Error encountered while installing @thegetty/quire-11ty@latest NPM package.
Error: ENOENT: no such file or directory, mkdir 'russell/node_modules/.install-npm-version-temp-nBqKav/node_modules/@thegetty/quire-11ty/node_modules'
```
I'm on a Mac (Apple silicon) running Ventura 13.4. npm version is 9.5.1 (error is the same using 9.6.7). 

I get the same errors with a different randomly generated temporary directory (.install-npm-version-temp-nBqKav) each time. The content directory gets generated in the new project, but nearly everything else is missing because the installer aborts on that error. Is there a node dependency that I'm missing or something else that anyone has encountered before?

**Comment by geealbers** [comment]
@mpopke I haven't encountered that issue before and in testing just now on my machine which is very close to yours (mac, apple silicon, ventura 13.2.1, npm 9.5.1, node 18.16.0, quire cli 1.0.0-rc.7) `quire new` worked. You're on node 18.16.0 as well I take it? I can ask our team here if they have any ideas. And perhaps someone else in the community will as well.

**Reply by mpopke** [reply, marked as answer]
Hi @geealbers,

I've been messing it with some more since yesterday, and I finally got Quire to work, but I had to install an nvm managed version of node to do it. Using the default node installer consistently produced the errors. I meant to update this thread sooner, but you know… museums.

Installing nvm, and using the nvm-managed version of 18.16.0 eventually worked. I can only imagine it's some broken dependency with the default node installation, probably caused by a security patch or xcode command line tools update between Ventura 13.2 and 13.4.

I don't know if others will have the same problem on later Mac OS versions, but it might be worth updating the Quire docs to prefer using nvm to manage node installations. I know that's less user-friendly, but it does seem more reliable overall (not just for Quire).
